app/components/modules/Header.jsx

```diff
@@ -113,7 +113,7 @@ class Header extends React.Component {
             page_title = `Stolen Account Recovery`;
         } else if (route.page === 'UserProfile') {
             user_name = route.params[0].slice(1);
-            page_title = user_name;
+            page_title = user_name.toLowerCase();
```

Comment:
Is this change necessary? I believe username will always be lowercase in the route